spylib/request.py

```diff
@@ -117,7 +140,34 @@ def make_service_request(
         **kwargs
     ):
         """
-        Cross service communication request, with an optional retry mechanism.
+        Makes a cross-service request with configurable timeouts and retry counts.
+
```

Comment:
the docs here are :fire:

Reply:
Thanks! I didn't add all docstrings yet - will do with time.